app/src/main/java/ru/firstSet/kotlinOne/FragmentMoviesDetails.kt

```diff
@@ -0,0 +1,33 @@
+package ru.firstSet.kotlinOne
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.fragment.app.Fragment
+
+class FragmentMoviesDetails : Fragment() {
+    private var imageViewPath2: View? = null
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? = inflater.inflate(R.layout.fragment_movies_details, container, false)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        imageViewPath2 = view.findViewById<View>(R.id.imageViewPath2).apply {
+            setOnClickListener {
+                activity?.supportFragmentManager?.beginTransaction()
```

Comment:
@kamar19 As I wrote before you can call onBakcPressed. No need to replace the fragment